scripts/template_config.yaml

```diff
@@ -0,0 +1,47 @@
+# This document is intended to serve as a template for constructing new
+# configuration YAMLs for use with the `simulation.py` script.
+
+# XXX do we want this to be an exhaustive template (all possible things
+# specified), or do we want this to be a minimal template?
+
+bda:
+    apply_bda: True
+    max_decorr: 0
+    pre_fs_int_time: !dimensionful 
+        value: 0
+        units: 's'
+    corr_FoV_angle: !dimensionful
+        value: 0
+        units: 'deg'
+    max_time: !dimensionful
+        value: 0
+        units: 's'
+    corr_int_time: !dimensionful
+        value: !!null
+        units: 's'
+defaults:
+    apply_defaults: False
+    default_config: 'h2c'
+filing:
+    outdir: '.'
+    outfile_name: 'quick_and_dirty_sim'
+    output_format: 'uvh5'
+    clobber: False
+    kwargs: {}
+# freq and time entries currently configured for hera_sim use
+freq:
+    n_freq: 1024
+    channel_width: 122070.3125
+    start_freq: 46920776.3671875
+time:
+    n_times: 10
+    integration_time: 8.59
+    start_time: 2457458.1738949567
+telescope:
+    array_layout: 'antenna_layout.csv'
+    # TODO: allow use with hera_sim.antpos
```

Comment:
flagging a TODO

Reply:
apparently already did this and hadn't committed the changes.